sdk/python/kfp/components/_components.py

```diff
@@ -276,3 +279,47 @@ def component_default_to_func_default(component_default: str, is_optional: bool)
     )
     task_factory.component_spec = component_spec
     return task_factory
+
+
+def _resolve_graph_task(graph_task: TaskSpec, graph_component_spec: ComponentSpec) -> TaskSpec:
+    from ..components import ComponentStore
+    component_store = ComponentStore.default_store
+
+    graph = graph_component_spec.implementation.graph
+
+    outputs_of_tasks = {}
+    def resolve_argument(argument):
+        if isinstance(argument, (str, int, float, bool)):
```

Comment:
Are we able to use the `PrimitiveTypes` type here?

Reply:
Unfortunately, no. Python complains that generic types cannot be used in `isinstance`.